tmpl/monitor.tmpl

```diff
@@ -1,6 +1,7 @@
 resource "datadog_monitor" "{{ .Id }}" {
   name               = "{{ .Name }}"
   type               = "{{ .Type }}"
+  {{ if .Tags }}tags               = {{ .Tags | commaSeparateList }}{{- end }}
```

Comment:
Wdyt about doing the same with built in functions? 
```
{{ range .Tags }}
{{ .... }}
{{ end }}
```

Reply:
Good point, I could have

```
tags = [
{{ range .Tags }}
{{ .... }},
{{ end }}
]
```

let me test it out